src/domUpdates.js

```diff
@@ -35,20 +35,38 @@ let domUpdates = {
            <h2 class="recipe-name" tabindex="0">${recipe.name}</h2>
          </article>
        </article>`
-      })
+    })
 
   },
 
-  displayRecipeInfo(recipe, ingredients) {
+  displayRecipeInfo(recipe,cookables,missinIng) {
+    let cookable; 
+    
+    let isRecipeCookable = cookables.find(recipe =>{
+      return recipe.name === recipe
+    }) !== undefined 
+
+ 
+    if (isRecipeCookable) {
+      
+      cookable = 'You can cook this recipe!'
+    } else {
+      cookable = `Can't cook this recipe! <br>
+      You are missing: <br>${this.formatObjectsToDisplatCorrectly((this.figureMissing(recipe, missinIng)))}` 
```

Comment:
I think a good refactor opportunity would be to have these invocations restructured.  Not sure how this could get done...but, it's a little hard to keep track of which values are getting passed in on line 55.